blocks_vertical/default_toolbox.js

```diff
@@ -580,6 +580,18 @@ Blockly.Blocks.defaultToolbox = '<xml id="toolbox-categories" style="display: no
         '</shadow>'+
       '</value>'+
     '</block>'+
+    '<block type="operator_contains" id="operator_contains">'+
+      '<value name="STRING1">'+
+        '<shadow type="text">'+
+          '<field name="TEXT">hello world</field>'+
```

Comment:
After looking at this in detail, I think we should switch this to `hello` and change the default for `STRING2` to `world`.